querybook/webapp/components/DataDocQueryCell/DataDocQueryCell.tsx

```diff
@@ -225,7 +223,7 @@ class DataDocQueryCellComponent extends React.PureComponent<IProps, IState> {
     }
 
     public get hasSamplingTables() {
-        return Object.keys(this.samplingTables).length > 0;
+        return Object.keys(this.state.samplingTables).length > 0;
```

Comment:
thanks for fixing it! could you help to add the sampleRate to the trackClick in the queryComposer as well? I forgot that

https://github.com/pinterest/querybook/blame/731a16635660ce1b0ad275c40abf266528b62410/querybook/webapp/components/QueryComposer/QueryComposer.tsx#L502